files/walle.py

```diff
@@ -535,9 +570,12 @@ def get_running_jobs(self, tool: str) -> list[Job]:
         self.conn.close()
         # Create a dictionary with job_id as key and object_store_id, and
         # update_time as values
-        if not running_jobs:
-            logger.warning("No running jobs with tool_id like '%s' found.", tool)
-            sys.exit(0)
+        if running_jobs:
+            logger.debug(f"Found {len(running_jobs)} running jobs matching '{tool}'")
+        else:
+            logger.debug(
+                f"No running jobs with tool_id like {tool} found.")
+            sys.exit(1)
```

Comment:
I would say not, but I think I changed this to `1` because it was somehow useful when running from cron. I can't remember why, and it doesn't seem to be useful now so have reverted to `sys.exit(0)`.